websockets.md

```diff
@@ -4,9 +4,9 @@ For use cases, when polling would be too resource consuming or not enough "real-
 
 ## Endpoint
 
-`[PlatformAddress]/ws/connector`
```

Comment:
Why not use `[WebSocketPlatformAddress]`

Reply:
I changed it right before your comment.